namespace/digest.go

```diff
@@ -40,7 +40,11 @@ func (d IntervalDigest) Hash() []byte {
 }
 
 func (d IntervalDigest) Bytes() []byte {
-	return append(append(d.min, d.max...), d.digest...)
+	return append(append(append(
+		make([]byte, 0, len(d.min)*2+len(d.digest)),
+		d.min...),
+		d.max...),
+		d.digest...)
```

Comment:
@adlerjohn will love this one 🤣 

Note that differently from the title, the nid does not get modified. Still the race detector would raise an alarm here before as this is seen as a WRITE.